sw/device/lib/dif/dif_pwrmgr.c

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+
+#include "sw/device/lib/common.h"
+
+#include "pwrmgr_regs.h"  // Generated.
```

Comment:
Can you add some kind of `#error "Power Manager DIF not implemented"` or the like? So if other people try to depend on this, they get a clearer error than just missing link-time dependencies on the function symbols?

Reply:
Good point, it may be a while before the implementation is done. Added `#error` in `dif_pwrmgr.c` and commented out the new definition in `meson.build` in order not to break the build (did not delete it because I will work on the implementation next, hope that's OK).

Reply:
SG!